Woof/Woof/Model/Owner/Owner.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+/**
+ A person who owns a pet (or several pets) and might be interested in finding a pet sitter for them.
+ */
+struct Owner: Identifiable, Codable {
+    /// Uniquely identifies the pet owner.
+    let id: UUID
+
+    /// The first name of the pet owner.
+    var name: String
+
+    /// The last name of the pet owner.
+    var surname: String
+
+    /// The phone number of the pet owner.
+    var phone: String
+
+    /// URL that contains an avatar image of the pet owner.
+    var avatarUrl: URL?
+
+    /// The address of the pet owner.
+    var address: String
+
+    /// The rating of the pet owner.
+    var rating: Double
+
+    /**
+     Creates an owner instance with the specified parameters with the unique identifier.
+
+     - Parameters:
+     - name: The first name of the pet owner.
+     - surname: The last name of the pet owner.
+     - phone: The phone number of the pet owner.
+     - avatarUrl: The URL that provides an avatar image of the pet owner.
+     - address: The address of the pet owner.
+     - rating: The rating of the pet owner.
+
+     The new owner instance has a unique id.
```

Comment:
Just nit. Maybe:

```suggestion
     Each instance of the new owner has a unique id.
```

Reply:
Thanks for your suggestion, added your version.